features/quizmaster/view_quiz_page.feature

```diff
@@ -5,7 +5,7 @@ Feature: As a Quizmaster
 
 Background:
   Given there is a quiz called "Trivia"
-  And there is a team named "Craft Academy"
+  And there is a team named "Craft Academy" playing "Trivia"
```

Comment:
Perhaps you should consider refactoring the step def for this to include setting the cookie (see below) and  minimizing exposure to what is going on in the background in the scenario? Perhaps it is not possible. Just a suggestion.